package.json

```diff
@@ -3,20 +3,49 @@
   "version": "0.1.0",
   "private": true,
   "description": "",
-  "workspaces": [
-    "packages/*"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "homepage": "https://github.com/MetaMask/template-snap-monorepo#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/template-snap-monorepo/issues"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/MetaMask/template-snap-monorepo.git"
   },
-  "author": "",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/MetaMask/template-snap-monorepo/issues"
+  "author": "",
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "build": "yarn workspaces foreach --parallel --topological --verbose run build",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
+    "start": "concurrently \"yarn start:site\" \"yarn start:snap\"",
+    "start:site": "yarn workspace site run start",
+    "start:snap": "yarn workspace snap run start",
+    "test": "echo \"TODO\""
```

Comment:
We should come up with some conventions for testing snaps, and add some example tests to this repo.

Reply:
Yes, other examples snaps and templates didn't had any tests. Planned to add those later on when we decide on how to test them.

Reply:
We will add this once we have an E2E testing tool for Snaps 😉

Reply:
#3